tests/platform_tests/test_platform_info.py

```diff
@@ -252,7 +252,7 @@ def test_turn_on_off_psu_and_check_psustatus(duthosts, enum_rand_one_per_hwsku_h
     psu_line_pattern = get_dut_psu_line_pattern(duthost)
 
     psu_num = get_healthy_psu_num(duthost)
-    if psu_num:
+    if psu_num is not None:
```

Comment:
When the psu_num become 0? I think the dut will be powered off if none of psu health

Reply:
In this testcase the API used to get PSU health is “psuutil status”. To determine the status this API tries to read the output power value which is not supported for the  PSU used in 7215 IXS-T1(M0). Hence the status is shown as “unknown” in this API. This corner case where power reading is not supported is not being handled correctly by the testcase.